lms/djangoapps/discussion/django_comment_client/tests/test_utils.py

```diff
@@ -1472,15 +1477,15 @@ def setUp(self):
 
     def test_discussion_division_disabled(self):
         course_discussion_settings = CourseDiscussionSettings.get(self.course.id)
-        assert {} == utils.get_group_names_by_id(course_discussion_settings)
+        assert {} == openedx.core.djangoapps.discussions.utils.get_group_names_by_id(course_discussion_settings)
```

Comment:
I will switch to using a direct import, but to explain how it's working here, we have `openedx.core.djangoapps.discussions.utils` at the top of this file, so that module is used here.